code/src/hooks/use-on-click-outside/useOnClickOutside.ts

```diff
@@ -0,0 +1,27 @@
+import { RefObject, useEffect } from "react";
+
+type Event = MouseEvent | TouchEvent;
```

Comment:
Avoid using `Event` name for custom type because it can create conflict with global `Event` interface, which is not ok. Try to came up with different name

Reply:
Renamed